ApollosProject/apollos-prototype
# Adds Cloudinary / Media resolver packages

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds a Cloudinary connector and moves the Media Resolvers into the `api-core`. The cloudinary connector is responsible for overwriting the default resolver and resolving image urls through cloudinary. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Boot the API
  - Query for images and ensure you are receiving cloudinary URLs
```
{
  node(id:"UniversalContentItem:7db601b238aa4345a818654300f83dd2") {
              ... on ContentItem {
            id
            images {
              sources {
                uri
              }
            }
          }
  }
}
```
2. Remove cloudinary connector. 
 - remove references to `Cloudinary` from data/index in `apollos-church-api`
 - make the same query above and ensure cloudinary is not in the result URL.



### What automated tests did you write?

Migrated existing tests

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
